Userland/Games/Chess/ChessWidget.cpp

```diff
@@ -522,9 +522,9 @@ void ChessWidget::playback_move(PlaybackDirection direction)
     update();
```

Comment:
Commit message: `depricated` should be `deprecated` (4x)

Also, you don't need to type "Summary:" before your commit description. Finally , please use your editor's word wrapping feature to automatically wrap lines after 72 characters.